packages/ai/router.py

```
@@ -1085,6 +1085,19 @@ async def _try_one_provider(
+                        try:
+                            from packages.ai.cost_tracker import record_usage as _record_cost
+                            _body = response.json()
+                            _usage = _body.get("usage") or {}
+                            _record_cost(
+                                model,
+                                provider_id=provider.provider_id,
+                                prompt_tokens=int(_usage.get("prompt_tokens") or 0),
+                                completion_tokens=int(_usage.get("completion_tokens") or 0),
+                            )
+                        except Exception:
+                            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)